build.gradle.kts

```diff
@@ -17,6 +21,7 @@ plugins {
     idea
     id(Deps.Plugins.detekt) version Versions.detekt
     id(Deps.Plugins.ktlint) version Versions.ktlint
+    //id(ProjectSettings.Templates.id) version ProjectSettings.version
```

Comment:
Napiš sem vysvětlení, nesmí se otagovat nová verze s tím, že by toto bylo odkomentované.

Reply:
A nedáme to pryč?

Reply:
Taky by to šlo dát pryč. Ale na druhou to chceš nějak otestovat, než uděláš commit, ne?